torchvision/prototype/transforms/functional/_color.py

```diff
@@ -2,9 +2,29 @@
 from torchvision.prototype import features
 from torchvision.transforms import functional_pil as _FP, functional_tensor as _FT
 
-from ._meta import get_dimensions_image_tensor
+from ._meta import _rgb_to_gray, get_dimensions_image_tensor, get_num_channels_image_tensor
+
+
+def _blend(image1: torch.Tensor, image2: torch.Tensor, ratio: float) -> torch.Tensor:
+    ratio = float(ratio)
+    fp = image1.is_floating_point()
+    bound = 1.0 if fp else 255.0
+    output = image1.mul(ratio).add_(image2, alpha=(1.0 - ratio)).clamp_(0, bound)
+    return output if fp else output.to(image1.dtype)
+
+
+def adjust_brightness_image_tensor(image: torch.Tensor, brightness_factor: float) -> torch.Tensor:
+    if brightness_factor < 0:
+        raise ValueError(f"brightness_factor ({brightness_factor}) is not non-negative.")
+
+    _FT._assert_channels(image, [1, 3])
+
+    fp = image.is_floating_point()
+    bound = 1.0 if fp else 255.0
+    output = image.mul(brightness_factor).clamp_(0, bound)
```

Comment:
Issue:

This clamp_ implementation is enforcing the histogram to take values between 0 and 1 in my case is not working the way I will expect since my images are normalized to values around 0 (ex: -0.2 to _0.8) so the multiplication with the factor of 1 will not return an identical image. 

Suggestion:
    ```output = image.mul(brightness_factor).clamp_(image.min(), image.max())```

Reply:
@MiChatz thanks for the feedback. There is (yet unwritten) assumption for color transformations on float images that image range is between [0, 1].